### DISCUSSION TITLE
Failed SU2 v7.5.1 compilation on Mac M1 Ventura

### DISCUSSION THREAD

**Discussion by fneri13** [su2code/SU2#1941] [Q&A]
Hello, 
I am trying to build from source the last official version of SU2 on my Mac M1 pro with OS Ventura. 
The first step works (./meson.py build), while the ninja command (./ninja -C build install) gives me the error attached in the err.txt file.
 
in particular the compilations seems to fail at this point: 
"ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
ninja: build stopped: subcommand failed."

Did someone ever had the same problem and knows how to solve it? (I have every compiler in place (gcc, g++, clang, mpicc, etc..)).

Thank you
[err.txt](https://github.com/su2code/SU2/files/10841387/err.txt)

**Comment by bigfooted** [comment]
Hi, 
Looks like a bunch of mpi errors, so probably related to the mpi installation. Maybe you can first try to compile without mpi and other options. If it is mpi, then check your installation of openmpi, mpich or whatever mpi you have.

If you are in Matteo's group at TUD, you can also directly ask @NAnand-TUD or @EvertBunschoten

**Reply by fneri13** [reply]
Thank you, it was indeed a problem with mpi. 
With "./meson.py build -Dwith-mpi=disabled" the compilation works.
